quetz/cli.py

```diff
@@ -278,11 +278,11 @@ def start(path: str,
 
     Parameters
     ----------
-    path : str
+    path: str
```

Comment:
I thought we'd agreed to keep the spaces before the `:`?

Reply:
Hi @benjamb 

You're right! I told you it is my own habit to write docstrings without whitespace-before-colon..

This only makes the built html having/not having thoses whitespaces, so we can just remove them everywhere if you prefer.

Reply:
I fixed the cli file. Thanks for having noticed that!
Let me know if you prefer to change the docstring style everywhere.

Reply:
I'm not that bothered myself tbh, though I agree with @wolfv that consistency is what's important.

Reply:
I also agree!